setup.py

```diff
@@ -6,7 +6,13 @@
 
 
 def readme():
-    with open('README.rst') as f:
+    import os.path
+    filename = 'README.rst'
+    if not os.path.exists(filename):
+        import inspect
+        script_path = os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe())))
+        filename = os.path.join(script_path, filename)
+    with open(filename) as f:
```

Comment:
By convention `python setup.py` is _always_ run from the directory it is in and so this is not needed. Many other things will break if this is not the case anyway.